app/controllers/home_controller.rb

```diff
@@ -7,9 +7,9 @@ def index
 
   def filtered_denounces
     if current_user&.postal_codes_by_place_of_interest&.any?
-      Denounce.for_postal_code(current_user.postal_codes_by_place_of_interest)
+      Denounce.includes(:address, :user, medias_attachments: :blob).for_postal_code(current_user.postal_codes_by_place_of_interest)
```

Comment:
you can remove some duplicated code with:
def filtered_denounces
  scope = Denounce.includes(:address, :user, medias_attachments: :blob)
  if current_user&.postal_codes_by_place_of_interest&.any?
    scope = scope.for_postal_code(....)
  else
    scope = scope.all
  end
end

Reply:
Thanks @softr8 , unfortunately we merged this before your comment but we can do your suggestion in other PR ;)